db/migrate/20190826161737_add_hide_followers_and_following_to_accounts.rb

```diff
@@ -0,0 +1,6 @@
+class AddHideFollowersAndFollowingToAccounts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :accounts, :hide_followers, :boolean
+    add_column :accounts, :hide_following, :boolean
```

Comment:
I am not entirely sure what these two separate options could be useful for. I understand that other software *could* use them separately, but it's probably a waste on our side. The accounts table is big already. So how about just one column? And transfer the local user's setting into it?

Reply:
Would adding two booleans instead of one really make a difference?
Using such booleans instead of the user setting for local accounts would make sense, but it would involve more work and a migration. I will investigate this, but I don't think that's a blocker for this PR, as any migration of this kind wouldn't be easier to do when introducing those columns than later down the road.

Reply:
Up